frontend/packages/ceph-storage-plugin/src/components/ocs-install/node-list.tsx

```diff
@@ -194,14 +194,27 @@ const CustomNodeTable: React.FC<CustomNodeTableProps> = ({ data, loaded, ocsProp
 
   const makeLabelNodesRequest = (selectedNodes: NodeKind[]): Promise<NodeKind>[] => {
     return selectedNodes.map((node: NodeKind) => {
-      const patch = [
-        {
-          op: 'add',
-          path: '/metadata/labels/cluster.ocs.openshift.io~1openshift-storage',
-          value: '',
-        },
-      ];
-      return k8sPatch(NodeModel, node, patch);
+      if (!node.metadata.labels['cluster.ocs.openshift.io/openshift-storage']) {
```

Comment:
create a getLabel selector in console/share folder.

Export https://github.com/openshift/console/blob/master/frontend/packages/ceph-storage-plugin/src/selectors/index.ts#L8

and then
`getLabels(node)[cephStorageLabel]`